CIP-0381/README.md

```diff
@@ -202,24 +316,28 @@ to perform the operations over these points (mults). Finally, when we want to ch
 one of the two points (or equivalently in this case we compute the conjugate), and multiply the result by the 
 other point. Only now we compute the final exponentiation
 and verify that the result is the identity element. In other words: 
-* the 'partial pairing' function, `BLS12_381_ppairing_ml` (stands for partial pairing miller loop) simply
+* the 'partial pairing' function, `bls12_381_millerLoop` simply
 computes the miller loop
-* the equality check function, `final_verify`, first
+* the equality check function, `bls12_381_finalVerify`, first
 computes an inverse, then a multiplication, a final exponentiation and checks that the element is the identity. 
 
+While the results of the miller loop are already elements in Fp12, they are not necessarily part of the group. This
+is why we call the type used in the built-ins `bls12_381_MlResult` rather than `bls12_381_GT`.
+
 Using the estimates exposed in the section `Costing of function`, we can see how this representation of
 GT elements is beneficial. Assume that we want to check the following relation: 
 `e(A,B) * e(C,D) =? e(E,F) * e(G,H)`. The miller look takes around 330us, and the final exponentiation
 around 370us. A full pairing would be 700us, and therefore checking this relation would cost around
 2,8ms. If, instead, we break down the pairing into a miller loop and a final exponentiation, and only
 compute the latter once, the cost of the relation above would be 330 * 4 + 370 = 1,7ms. 
-                                         
-For this reason it is important to limit what can be done with GTElements, as the pairing really is not the full 
+
+For this reason it is important to limit what can be done with `bls12_381_MlResult`, as the pairing really is not the full
 pairing operation, but only the miller loop. 
 ### Source implementation
 * [BLST](https://github.com/supranational/blst) library, providing the algebraic operations.
-* [cardano-base](https://github.com/input-output-hk/cardano-base/tree/bls12-381/cardano-crypto-class/src/Cardano/Crypto/EllipticCurve)
+* [cardano-base](https://github.com/input-output-hk/cardano-base/tree/master/cardano-crypto-class/src/Cardano/Crypto/EllipticCurve)
 with the haskell FFI to the BLST library.
+* [plutus](https://github.com/input-output-hk/plutus/pull/5231)
 
 Other libraries of interest
 * [Ethereum support for BLS12_381](https://eips.ethereum.org/EIPS/eip-2537). Not directly relevant as this is an 
```

Comment:
"Comparison with existing function" -> "Comparison with existing functions" on line 346.

"there exists proofs" -> "there exist proofs" on line 356.

"BLS12 curve" -> "The BLS12 curve" on line 367?   Also, is it the BLS12 curve or the BLS12-381 curve (I'm not sure)?